process/heartbeat/interceptedPeerAuthentication.go

```diff
@@ -136,6 +195,25 @@ func (ipa *interceptedPeerAuthentication) String() string {
 	)
 }
 
+func (ipa *interceptedPeerAuthentication) verifyPayload() error {
+	payload := &heartbeat.Payload{}
+	err := ipa.marshalizer.Unmarshal(payload, ipa.peerAuthentication.Payload)
+	if err != nil {
+		return err
+	}
+
+	currentTimeStamp := uint64(time.Now().Unix())
```

Comment:
suggestion to simplify:
```GO
    currentTimeStamp := time.Now().Unix()
    messageTimeStamp := payload.Timestamp
    minTimestampAllowed := currentTimeStamp - ipa.expiryTimespanInSec
    maxTimestampAllowed := currentTimeStamp + payloadExpiryThresholdInSec
    if messageTimeStamp < minTimestampAllowed || messageTimeStamp > maxTimestampAllowed {
        return process.ErrMessageExpired
    }
```
and make the Payload.Timestamp from the proto file an int64